app/views/collections/_form.html.erb

```diff
@@ -1,9 +1,19 @@
 <%= simple_form_for @form, html: { class: 'editor' } do |f| %>
   <div id="descriptions_display">
-    <h2 class="non lower">Descriptions</h2>
-    <% f.object.rendered_terms.each do |term| %>
-      <%= render_edit_field_partial term, f: f %>
-    <% end %>
+    <h2 class="non lower">Metadata</h2>
```

Comment:
I wasn't even thinking of the tabs, but I might like to see those change as well. I was talking about the `<h2>Metadata</h2>` in the work show page:

https://github.com/projecthydra/sufia/blob/master/app/views/curation_concerns/base/_metadata.html.erb#L1

Which is set here:

https://github.com/projecthydra/sufia/blob/master/config/locales/sufia.en.yml#L344